website/astro.config.mjs

```diff
@@ -1,4 +1,8 @@
 import { defineConfig } from 'astro/config';
 
 // https://astro.build/config
-export default defineConfig({});
+export default defineConfig({
+  // setup for github pages
+  site: 'https://pheno-agency.github.io',
+  base: '/proxy-memoize',
```

Comment:
@Sajad-Sharhani I think the base should be /website, it's the directory that the site lives in. And please add the github workflow that astro recommends.